packages/outstatic/src/utils/useOid.tsx

```diff
@@ -15,19 +15,22 @@ const useOid = () => {
     fetchPolicy: 'no-cache'
   })
 
-  const fetchOid = async () => {
+  const fetchOid = useCallback(async () => {
```

Comment:
Memoized via callback hook. Prevent neverending calls if `fetchOid` was used as a dependency in another component.